src/main/resources/i18n/messages.properties

```diff
@@ -1968,6 +1968,18 @@ SampleDetails.description=Description
 SampleDetails.organism=Organism
 SampleDetails.createdDate=Created
 SampleDetails.modifiedDate=Modified
+
+# ========================================================================================== #
+# ADD SAMPLE REACT COMPONENT                                                                 #
+# ========================================================================================== #
+AddSample.title=Create New Sample in Project
+AddSample.name=Sample Name
+AddSample.organism=Organism
+AddSample.submit=Create Sample
+server.AddSample.error.length=Sample name must have at least 3 characters.
```

Comment:
This needs to be updated. Currently for example if we set the name to SAM it still displays this error but it has 3 characters

Reply:
Updated in 9e3f14a1dd4531bd26770761f537588e6690254e